src/hypothesis/core.py

```diff
@@ -1157,8 +1148,7 @@ def template_condition(data):
             list(runner.interesting_examples.values())[0].buffer)
         with BuildContext(data):
             return data.draw(search)
-    if (
-        runner.valid_examples <= settings.min_satisfying_examples and
+    if runner.valid_examples == 0 and (
         runner.exit_reason != ExitReason.finished
```

Comment:
Is this clause ever active?

Reply:
Yep - various things in `tests/pytest/test_capture.py` fail if it's changed.